reminder/module.py

```diff
@@ -167,7 +161,7 @@ async def _send_reminder_list(self, ctx, query):
                 "remind_name": _(ctx, "Reminded"),
                 "remind_date": _(ctx, "Remind date"),
                 "status": _(ctx, "Status"),
-                "url": "https://discord.com/channels/",
+                "url": "URL stub",
```

Comment:
Great, and please use `_(ctx, "URL stub")`.

Reply:
Done.